### PR TITLE
Seismic modelling tutorial

### PR DESCRIPTION
**Pull request by mloubout**
A modelling notebook tutorial

### CODE REVIEW

**Comment by FabioLuporini** [comment]
Assignees : please take a look and comment

**Comment by navjotk** [comment]
Lots of spelling and grammar that can probably be corrected later, but otherwise this is great!

**Comment by mloubout** [comment]
> Lots of spelling and grammar 

Yep, was expected, english not as easy than coding, specially doing it quick :D

**Comment by FabioLuporini** [comment]
@ggorman Can you take a look at this please

**Comment by mloubout** [comment]
This needs some more work and (english) cleanup from my side. It is still a bit vague in some point, but open to suggestion to have a proper introductory tutorial.

**Comment by FabioLuporini** [comment]
How's this coming up?

**Comment by mloubout** [comment]
Slowly getting there

**Comment by mloubout** [comment]
@mlange05 @navjotk @ggorman 

I still need to go over the english, phrasing and stuff. Can you just let me know if you feel that I left some details out you think should be in it.

**Comment by mlange05** [comment]
Ok, I've reorganised a few things and edited some of the text in the tutorial notebook. I've also factored the plotting into separate utilities for later re-use and added travis testing. I think there is still a little bit more we want to say about the actual construction of the expressions, in particular time indices for source/rec, but we're getting close I would say.

@mloubout can you please take a look and see if you are happy with my changes? I'm happy to revert things if not...

File: examples/seismic/tutorials/test_01_modelling.ipynb [hunk 1]
```diff
@@ -261,7 +260,7 @@
    },
    "outputs": [],
    "source": [
-    "# In order to represent the wavefield u, and the squre slowness we need symbolic object \n",
+    "# In order to represent the wavefield u, and the sqaure slowness we need symbolic object \n",
```
Comment (mlange05) [inline examples/seismic/tutorials/test_01_modelling.ipynb]: Fixing a typo with a typo...? :wink:

Comment (FabioLuporini) [inline examples/seismic/tutorials/test_01_modelling.ipynb]: Also, "In order to represent the wavefield u and the square slowness, ..."

**Comment by mloubout** [comment]
@mlange05 Looks very good overall, I added a tiny section on src/rec times.

**Comment by mlange05** [comment]
Yes, looks great now. Good to go from me.

**Comment by mloubout** [comment]
Can we also link it to the website (the notebook). Not sure how do do that.

**Comment by mlange05** [comment]
> Can we also link it to the website (the notebook). Not sure how do do that.

> EDIT: And remove the previous tutorial on seismic that is outdated (like 6 month outdated)

That was the plan! I'll look into the logistics of how to do that.

**Comment by mlange05** [comment]
Anybody else want to take a look at this before merging? I think this is good to go, so please speak up or I'll merge once travis is happy. @ggorman @philippwitte

**Comment by mloubout** [comment]
` ipython nbconvert test_01_modelling.ipynb --to rst` and we have auto documentation from the notebook. Should we add it here?

**Comment by mlange05** [comment]
I was more thinking of using `nbviewer.jupyter.org`, so that we don't have to care about auto-converting the notebooks all the time. I think for that we need to merge the notebook first though. So, can we leave the docs for another PR (possibly just on the website)? If yes, I'll remove the last commit before merging.

**Comment by mloubout** [comment]
Works too. Let's revert this then, but let's make sure we get this done this week so that we can point people to it in Paris.

**Comment by mlange05** [comment]
Great, merged. To view via `nbviewer` we can now simply use weblinks like [this](http://nbviewer.jupyter.org/github/opesci/devito/blob/master/examples/seismic/tutorials/test_01_modelling.ipynb). I'll update the website soon.

**Pull request merged**